Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "heroku-ruby-buildpack"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+libcnb = "0.6.0"
+tempfile = "3.3.0"
+flate2 = "1"
+ureq = "2.2.1"
```

Comment:
```suggestion
ureq = "2.4.0"
```

Reply:
If it helps, a quick way to keep these up to date is to use `cargo upgrade` (note: different from `cargo update`), which is part of this plugin:
https://github.com/killercup/cargo-edit#cargo-upgrade